examples/aws/cognito-rds-s3/README.md

```diff
@@ -111,16 +111,10 @@ Follow the [Configure Katib](../rds-s3/README.md#3-configure-katib) section from
 
         # Envoy filter
         kustomize build distributions/aws/aws-istio-envoy-filter/base | kubectl apply -f -
-        
-        # Configured for AWS RDS and AWS S3
```

Comment:
nit: afaik, @rrrkharse mentioned something related to ordering because of which he kept these steps at the end
also for uniformity, let's keep modified components after vanilla

can you move this section above Ingress?

generic components
RDS,S3
Cognito

Same for here - https://github.com/awslabs/kubeflow-manifests/blob/v1.3-branch/examples/aws/cognito-rds-s3/kustomization.yaml. I just noticed the ordering in this file is also reverse

Reply:
Done first two parts. 

I like the idea of having the order consistent. But I am afraid on making changes which might break at the last min. we know authentication works in RDS_s3 side so let it be for now...we can fix the order later.